README.md

```diff
@@ -19,8 +19,60 @@ There are no complicated dependencies. All you need is [Node.js](https://nodejs.
 and [Yarn](https://yarnpkg.com/en/) installed, everything else will be installed
 from NPM.
 
+## Table of Contents
+
+* [Requirements](#requirements)
+* [How to install](#how-to-install)
+* [Where to start](#where-to-start)
+* [Technologies used](#technologies)
+
+## Requirements
+
+This project requires the following to be installed.
+
+* [Node.js 8+](https://nodejs.org/en/)
+* [Yarn](https://yarnpkg.com/en/docs/install)
+
+## How to Install
+
+If you have everything from [Requirements](#requirements) then you can simply run: `yarn`.
+
+This project uses [Yarn Workspaces](https://yarnpkg.com/lang/en/docs/workspaces/) so all the project dependencies for all the folders will be installed automatically.
```

Comment:
Could you please also update the installation instructions in packages? For example, this is confusing: https://github.com/kiwicom/jsweekend/tree/master/start#installation (seems like I need to install dependencies again)

Reply:
It's coming as a second PR.